{
  "contexts": {
    "device": { "arch": "amd64", "num_cpu": 8 },
    "os": { "name": "linux" },
    "runtime": {
      "go_maxprocs": 8,
      "go_numcgocalls": 1,
      "go_numroutines": 2,
      "name": "go",
      "version": "go1.12.17"
    }
  },
  "event_id": "af536c31b68a473f97e579507ce155e3",
  "level": "error",
  "platform": "go",
  "sdk": {
    "name": "sentry.go",
    "version": "0.6.0",
    "integrations": [
      "ContextifyFrames",
      "Environment",
      "IgnoreErrors",
      "Modules"
    ],
    "packages": [{ "name": "sentry-go", "version": "0.6.0" }]
  },
  "server_name": "7337a9a27ac6",
  "timestamp": "2020-04-25T19:36:56.584908333Z",
  "user": {},
  "exception": [
    {
      "type": "*os.PathError",
      "value": "open ./test.txt: no such file or directory",
      "stacktrace": {
        "frames": [
          {
            "function": "main",
            "module": "main",
            "filename": "file-not-found.go",
            "abs_path": "/app/file-not-found/file-not-found.go",
            "lineno": 25,
            "pre_context": [
              "",
              "\tdefer sentry.Flush(2 * time.Second)",
              "",
              "\tf, err := os.Open(\"./test.txt\")",
              "\tif err != nil {"
            ],
            "context_line": "\t\tsentry.CaptureException(err)",
            "post_context": [
              "\t\tlog.Printf(\"reported to Sentry: %s\", err)",
              "\t\treturn",
              "\t}",
              "\tfmt.Println(f.Name(), \"opened successfully\")",
              "}"
            ],
            "in_app": true
          }
        ]
      }
    }
  ]
}
